### PR TITLE
Bug 1280230: Fix upgrade playbook version check with multiple masters.

### PR DESCRIPTION
**Pull request by abutcher**
https://bugzilla.redhat.com/show_bug.cgi?id=1280230

#839 also fixes this problem

### CODE REVIEW

File: playbooks/adhoc/upgrades/upgrade.yml [hunk 1]
```diff
@@ -156,7 +156,7 @@
         to_version: '3.1'
         role: master
         config_base: "{{ hostvars[inventory_hostname].openshift.common.config_base }}"
-      when: deployment_type in ['openshift-enterprise', 'atomic-enterprise'] and g_aos_versions.curr_version | version_compare('3.1', '>=')
+      when: deployment_type in ['openshift-enterprise', 'atomic-enterprise'] and hostvars[groups.masters.0].g_aos_versions.curr_version | version_compare('3.1', '>=')
```
Comment (detiber) [inline playbooks/adhoc/upgrades/upgrade.yml]: Why not just use g_new_version here?

Comment (abutcher) [inline playbooks/adhoc/upgrades/upgrade.yml]: g_new_version is also tied to the first master but it should probably be used instead

Comment (detiber) [inline playbooks/adhoc/upgrades/upgrade.yml]: We can update the version checking to run against all hosts...  that would also validate that they have the repo available on all of the hosts as well.

Comment (detiber) [inline playbooks/adhoc/upgrades/upgrade.yml]: Should just be a matter of changing:
```
- name: Perform upgrade version checking
  hosts: masters[0]
```
to:
```
- name: Perform upgrade version checking
  hosts: masters nodes
```

**Comment by abutcher** [comment]
Obviated by #839

**Pull request closed without merging**